### DISCUSSION TITLE
Buttonless: a cleaner way to play iSSB on PC

### DISCUSSION THREAD

**Discussion by Nightcaat** [lufinkey/iSSB#31] [Show and tell]
[Buttonless](https://github.com/Nightcaat/iSSB/tree/buttonless) is a fork of iSSB that simply sets the scale of the on-screen controller buttons to 0. It also removes the control options button from the options menu as it is useless on platforms controlled by the keyboard and mouse. This is just a quick thing I hacked together to get the buttons out of the way when playing with my friend over [Parsec](https://parsec.app), so you should probably not try using it on mobile.

The ideal solution would be to:
* Add a button opacity slider (maybe one for each button if you wanna be REAL fancy, no need for me to have the start button visible all the time for example)
* Show the buttons if touch input is detected (and make it work on Windows and Linux, though I have no clue if they handle multi-touch differently than iOS and Android)
* Hide the buttons if keyboard input is detected (and make it work on other platforms, no shame if people wanna play on an iPad Pro keyboard or whatever)

That is beyond my capabilities at the moment but I would like to work toward that someday.

Quick video showcase:
[![YouTube video thumbnail showing the iSSB control options and on-screen controller buttons removed with the name Buttonless on top](https://i.ytimg.com/vi/QhILkwn9u8A/maxresdefault.jpg)](https://youtube.com/watch?v=QhILkwn9u8A)

**Comment by lufinkey** [comment]
This is great! And makes the screen recordings look a lot nicer too. If it was an option instead of by default, I'd happily accept it as a pull request
